pkg/hyperkit/driver.go

```diff
@@ -100,7 +100,7 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	b2dutils := mcnutils.NewB2dUtils(d.StorePath)
+	b2dutils := mcnutils.NewB2dUtils(d.StorePath, "")
```

Comment:
force the name `crc.disk` so we do not have to rely on the default value. (Note: i'd rather see the default disappear as it is too specific to a driver implementation)

Reply:
The following: 

https://github.com/code-ready/machine/blob/b6d974ad44d01f18861a8ad3c0949cfd56b15032/libmachine/mcnutils/crc.go#L172-L174

might have to be removed as it is unwanted behaviour.

Reply:
@gbraad I agree but that will be in a separate commit message, we have to release the hyperkit with latest update as of now, in future we should do that.